tailcfg/tailcfg.go

```diff
@@ -483,6 +483,8 @@ func (hi *Hostinfo) TailscaleSSHEnabled() bool {
 	return hi != nil && len(hi.SSH_HostKeys) > 0
 }
 
+func (v HostinfoView) TailscaleSSHEnabled() bool { return v.ж.TailscaleSSHEnabled() }
```

Comment:
Ah, thank you! I figured there was more places to add this, but wasn't sure what they were.

Reply:
Will move this to a separate commit, it's likely we'll want this sooner

(I checked with Maisem that the views code gen tool does not handle custom methods)